include/olg_scheduler/scheduler.hpp

```diff
@@ -18,12 +18,12 @@
 #pragma once
 
 #include <cavl/cavl.hpp>
```

Comment:
The idiomatic inclusion pattern for cavl is just cavl.hpp. The current form will break if the user decided to submodule the cavl repo. The include path settings will need to be adjusted.

```suggestion
#include <cavl.hpp>
```